### DISCUSSION TITLE
Version changes since 7.2.1

### DISCUSSION THREAD

**Discussion by hobipivo** [universam1/iSpindel#609] [General]
Hi Sam,
can you please document the version changes since 7.2.1?
Greetings Christian

**Comment by thegreatgunbantoad** [comment]
If you go to the main page and find the bit next to master that says '51 tags'  
![image](https://github.com/universam1/iSpindel/assets/60237779/82439d9a-1c16-4372-a557-78b18cb80f86)
Click that and click 7.2.1 then it will show you the 10 commits difference to the master branch. 
![image](https://github.com/universam1/iSpindel/assets/60237779/b414aa2b-ec30-4526-a11f-8ef85dc55824)

You should get this (click below link):
https://github.com/universam1/iSpindel/compare/7.2.1...master

Part if the beauty of git is that it is sort of self documenting in this way.

**Reply by thegreatgunbantoad** [reply]
Not sure how you do a diff between specific pairs of branches/commits/tags in github.  Need to figure that one out.  But if you have a git archive on a local machine git lens in VS Code does a great job.

**Comment by hobipivo** [comment]
Thanks thegreatgunbantoad for help and information. 
It would be nice if someone could document the changes (What's Changed) under "(https://github.com/universam1/iSpindel/releases)" and (ChangeLog) under "(http://iSpindel.de)".

**Reply by thegreatgunbantoad** [reply]
I'm sure @universam1 won't mind you wrting the text for him to copy and paste in :)

Since there are over 2 years worth of changes (42 commits over 61 files), I think I'd just be inclined to write something along the lines of:
Firmware 7.3.3: For changes between version Firmare 7.1.0 and Firmware 7.3.3 please see the following git compare: https://github.com/universam1/iSpindel/compare/7.1.0...7.3.3
